### PR TITLE
Makes 0 a define

### PR DESCRIPTION
**Pull request by monster860**
## About The Pull Request

This pull request replaces 0 with a constant define.

## Why It's Good For The Game

I've heard that more constants is better for the game. Making 0 a constant will improve the readability of the code, and make it so that if someone needs to replace all the zeros with something other than 0 its much easier and does not involve a massive find-replace.

## Changelog
:cl:
tweak: 0 is now a constant
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
what the fuck

**Comment by Hopekz** [comment]
> what the fuck

new rebase you wouldn't understand

**Comment by LemonInTheDark** [comment]
My man has reached the peak of earth. I didn't think this level of based was humanly possible.

**Comment by monster860** [comment]
the CI build passed. Speedmerge now.

**Review by ZeWaka — APPROVED** [review]
Very good monster, I like it.

**Comment by MCterra10** [comment]
ngl this is pretty based

**Comment by ArcaneMusic** [comment]
I'm scared, I'm terrified, I'm shaking.
Ya know how there's that classic joke about "I was good at math until math started using more letters than numbers?"
This is us now.

**Comment by LemonInTheDark** [comment]
Did... did you do this with regex? Please say yes.

**Comment by MrPerson** [comment]
Why did you stop at 0? There's 1, 2, 3, 4, and even the dreaded 6.

**Comment by monster860** [comment]
No, I manually went through each 10,000 zeros and replaced them by hand.

**Comment by LemonInTheDark** [comment]
That's good, I was worried it might have caught some commented code.

File: code/__DEFINES/atmospherics.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #define TCMB					2.7
 /// -48.15degC
 #define TCRYO					225
-/// 0degC
+/// ZEROdegC
```
Comment (Jalleo) [inline code/__DEFINES/atmospherics.dm]: Um?

**Comment by trollbreeder** [comment]
God is dead and we have killed him

**Pull request closed without merging**